Makefile

```
@@ -70,38 +68,24 @@ docker-publish: docker-publish-dsp-api-image docker-publish-sipi-image ## publis
 docker-image-tag: ## prints the docker image tag
 	@$(SBTX) -Dsbt.log.noformat=true -Dsbt.supershell=false -Dsbt.ci=true -error "print dockerImageTag"
 
-#################################
-## Docker-Compose targets
-#################################
-
-.PHONY: print-env-file
-print-env-file: ## prints the env file used by dsp-stack
-	@cat .env
-
-.PHONY: env-file
-env-file: ## write the env file used by dsp-stack.
-	@echo DOCKERHOST=$(DOCKERHOST) > .env
-	@echo KNORA_DB_REPOSITORY_NAME=$(KNORA_DB_REPOSITORY_NAME) >> .env
-	@echo LOCAL_HOME=$(CURRENT_DIR) >> .env
-
 #################################
 ## DSP Stack Targets
 #################################
 
 .PHONY: stack-up
-stack-up: docker-build env-file ## starts the dsp-stack: fuseki, sipi, api and app.
+stack-up: docker-build ## starts the dsp-stack: fuseki, sipi, api and app.
 	@docker compose -f docker-compose.yml up -d db
 	$(CURRENT_DIR)/webapi/scripts/wait-for-db.sh
 	@docker compose -f docker-compose.yml up -d
 	$(CURRENT_DIR)/webapi/scripts/wait-for-api.sh
 
 .PHONY: stack-up-fast
-stack-up-fast: docker-build-dsp-api-image env-file ## starts the dsp-stack by skipping rebuilding most of the images (only api image is rebuilt).
+stack-up-fast: docker-build-dsp-api-image ## starts the dsp-stack by skipping rebuilding most of the images (only api image is rebuilt).
 	docker-compose -f docker-compose.yml up -d
 
 .PHONY: stack-up-ci
-stack-up-ci: KNORA_DB_REPOSITORY_NAME := knora-test-unit
-stack-up-ci: docker-build env-file print-env-file ## starts the dsp-stack using 'knora-test-unit' repository: fuseki, sipi, api.
+stack-up-ci: KNORA_WEBAPI_TRIPLESTORE_FUSEKI_REPOSITORY_NAME := knora-test-unit
```

```suggestion
stack-up-ci: KNORA_DB_REPOSITORY_NAME := knora-test-unit
```

**question**: Shouldn't this remain the old environment variable name?

---

I would hope that no, but I'm glad to have someone checking my logic.  
Here's my reasoning:

Previously, it worked as follows:

- in `vars.mk`, we had `KNORA_DB_REPOSITORY_NAME := knora-test`
- in any other make target case, this remained unchanged; here in `stack-up-ci`, this was overwritten with `KNORA_DB_REPOSITORY_NAME := knora-test-unit`; if the stack was started without make, `KNORA_DB_REPOSITORY_NAME` was not set.
- in `docker-compose.yml` we had `KNORA_WEBAPI_TRIPLESTORE_FUSEKI_REPOSITORY_NAME=${KNORA_DB_REPOSITORY_NAME}`, so the same applied, the variable there could be "knora-test", "knora-test-unit" or undefined.
- in `application.conf` we have another default `repository-name = "knora-test"` `repository-name = ${?KNORA_WEBAPI_TRIPLESTORE_FUSEKI_REPOSITORY_NAME}`

My thinking now was:
- the default "knora-test" in vars.mk is redundant, in that case we can just make sure that it remains undefined so that the same default is used in application.conf.
- in that case, the only value that `KNORA_DB_REPOSITORY_NAME` can ever have, is "knora-test-unit" and the only point where it's read is in the docker-compose file where it is written into `KNORA_WEBAPI_TRIPLESTORE_FUSEKI_REPOSITORY_NAME`, so we could use this variable instead and have the same behaviour.

Does my thinking make any sense?

---

as discussed